students/hongju/users/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+# Create your models here.
+class User(models.Model):
+    name            = models.CharField(max_length=45)
+    email           = models.CharField(max_length=300)
+    password        = models.CharField(max_length=45)
+    phone_number    = models.CharField(max_length=45)
+    additional_info = models.TextField()
```

Comment:
```suggestion
    phone_number    = models.CharField(max_length=45)
```
1. 그 외 정보는 user 테이블에 추가할 만한 사항이 있으면 추가 하라는 의미 입니다.
2. 데이터의 이력 관리를 위해서 created_at, updated_at column을 추가해주세요.

```python
created_at = models.DateTimeField(auto_now_add=True)
updated_at = models.DateTimeField(auto_now=True)
```